src/main/kotlin/step4/racingcar/racingcar/CarRacing.kt

```diff
@@ -0,0 +1,26 @@
+package step4.racingcar.racingcar
+
+object CarRacing {
```

Comment:
혹시, 게임 중에 그란투리스모나 카트라이더를 해보신 적이 있을까요? 둘 다 자동차 게임이지요!
일단, 로비에서는 참여자를 받고, 맵을 고릅니다. 그 맵 안에서는 몇 바퀴 돌아야 되는지에 대한 설정이 들어가 있지요.

그러고, 게임이 시작하면, 참여자는 맵을 완주하는 게 목표가 되고, 각 참여자가 완주한 횟수를 게임에서는 기록을 하고, 기록이 끝나면 그 기록을 통하여 1,2,3등을 고르게 돼요.

동윤님과 저와 구현하고 있는 자동차 게임이 그런 원리이지 않을까요?

Main은 로비가 되고, CarRacing은 진행 된 게임을 의미하게 될거에요. 그리고, Main에서는 게임의 조작을 플레이어가 할 것인지? 자동으로 할 것인지? 에 대해서도 설정을 할 수 있을 것 같아요!

```suggestion
class CarRacing(private val racers: List<String>, moveCount: Int) {
```

이런 부분을 다시 생각해서, CarRacing을 object class가 아닌, class로 변경하여, racer와 moveCount를 가지고 있도록 리팩토링해볼까요?

Reply:
[좋은 객체의 7가지 덕목](https://codingnuri.com/seven-virtues-of-good-object/) 에 대한 글도 읽어주시길 바래요!

Reply:
예시를 듣고나니 감을 잡은거 같은데 다시 한번 수정 해보겠습니다~